sklearn/decomposition/pca.py

```diff
@@ -237,6 +237,14 @@ class PCA(_BasePCA):
         n_components, or the lesser value of n_features and n_samples
         if n_components is None.
 
+    n_features_ : int
+        The number of features in the data matrix the PCA transformer was
+        fitted on.
+
+    n_samples_ : int
+        The number of samples in the data matrix the PCA transformer was
```

Comment:
Maybe just "Number of samples in the training data"?
Is this always present?

Reply:
Thank you. I still need to improve my English...